### ISSUE TITLE
Make a video tutorial

### ISSUE THREAD

**Issue by Mholsen** [MicrosoftDocs/powerbi-docs#597]
Hi,
You should make a video tutorial going through all the steps mentioned here, or get Guy in a Cube to do it.
Also, I really think that you should split these tutorials into multiple one depending on the system (eg. SAP BW or HANA etc.). Having them both/all in one guide adds to the confusion, and right now it's hard to follow along with all the jumping around.

---
#### Document details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: e8d4509b-ab31-a917-16a2-8ed7ce834c41
* Version Independent ID: 31f21190-cd0d-f369-fc4a-48ca7ac1ccb0
* Content: [Use Kerberos for single sign-on (SSO) to on-premises data sources - Power BI](https://docs.microsoft.com/en-gb/power-bi/service-gateway-sso-kerberos#configuring-kerberos-constrained-delegation-for-the-gateway-and-data-source)
* Content Source: [powerbi-docs/service-gateway-sso-kerberos.md](https://github.com/MicrosoftDocs/powerbi-docs/blob/live/powerbi-docs/service-gateway-sso-kerberos.md)
* Service: **powerbi**
* GitHub Login: @mgblythe
* Microsoft Alias: **mblythe**

**Comment by mgblythe** [comment]
Thanks @Mholsen, we're planning some improvements to the gateway docs, and I'll keep this feedback in mind. @arthiriyer as FYI.

**Comment by mgblythe** [comment]
We've just done an update on the rest of the gateway docs and we're now planning more work on the SSO docs. That said there are no plans for a video, so closing this one out. Thanks!
